source/slang/slang-ast-base.h

```diff
@@ -4,7 +4,7 @@
 
 #include "slang-ast-support-types.h"
 
-#include "slang-ast-generated.h"
+#include "slang-generated-ast.h"
```

Comment:
Not very important, but if we ever change these names again, I might argue for a blanket policy of having names like `slang-ast.generated.h` so that files of the form `*.generated.*` always represent generated code and then we can set up our `.gitignore` and things like that to always ignore them.

(In fact, I suspect we *do* already have `.gitignore` lines that ignore `*-generated.h`, so we might want to make sure that this PR updates those)

Reply:
Ok fixed .gitignore - which was actually already out of date.